website/index.html

```diff
@@ -236,10 +236,10 @@
               <!--</div>-->
             <!--</a>-->
           <!--</div>-->
-          <div class="offset-md-2 col-lg-8 col-md-10 col-sm-12 register-btn" style="left: 8vh;">
+          <div class="offset-md-2 col-lg-8 col-md-10 col-sm-12 register-btn" >
             <div class="row">
               <a href="https://app.hackpsu.org/signup">
-                <img src="./assets/images/planet7small.svg" style="height: 16vh; width: auto; position: relative;left: -50;bottom: 32"/>
+                <img src="./assets/images/planet7small.svg" style="height: 15.2vh; width: auto; position: relative;left: -60;bottom: 25"/>
```

Comment:
In terms of changes I would suggest looking at the <div> line rather than the <img> line. The img tag just controls the planet itself and it might be useful to investigate if it looks better to move the position of the _entire__ button (controlled by the div tags) rather than just the planet